postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -65,7 +65,7 @@ static dotResult exec_command_set(BteqScanState scan_state, bool active_branch);
 static void ignore_dot_options(BteqScanState scan_state);
 static bool is_branching_command(const char *cmd);
 static char *cat_space(char *str);
-
+static char *prompt_for_password(const char *username);
```

Comment:
Can't find the usage of this function. I wonder why we need this here.

Reply:
i deleted it